### PR TITLE
First version of fr.po

### PR DESCRIPTION
**Pull request by ImperatorS79**
For some string I wasn't sure of the meaning, and while we won't be able to change the language used in POL5, I won't be able to verify.

But it's better thatn nothing :)

### CODE REVIEW

**Comment by plata** [comment]
@qparis could you review this? I don't know French.

File: i18n/fr.po [hunk 1]
```diff
@@ -0,0 +1,615 @@
...
+
+#: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java:137
+#, java-format
+msgid "This installation program is provided by: {0}"
+msgstr "Cet installeur est fourni par: {0}"
+
+#: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java:138
+#, java-format
+msgid "{0} will be installed in: {1}"
+msgstr "{0} sera installé dans: {1}"
```
Comment (qparis) [inline i18n/fr.po]: Please add a space before ":"

File: i18n/fr.po [hunk 2]
```diff
@@ -0,0 +1,615 @@
...
+msgid ""
+"{0} is not responsible for anything that might happen as a result of using "
+"these scripts."
+msgstr ""
+"{0} n'est pas responsable de ce qui pourrait se produire en utilisant ces "
+"scripts."
+
+#: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java:141
+msgid "Click \"Next\" to start."
+msgstr "Cliquer sur 'Suivant\" pour commencer."
```
Comment (qparis) [inline i18n/fr.po]: Choose either \"Suivant\" or 'Suivant'

File: i18n/fr.po [hunk 3]
```diff
@@ -0,0 +1,615 @@
...
+
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerInformationTab.java:38
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerInformationTab.java:50
+msgid "Information"
+msgstr "Information"
+
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerInformationTab.java:58
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/AboutPanel.java:69
+msgid "Name:"
+msgstr "Nom:"
```
Comment (qparis) [inline i18n/fr.po]: Please add a space before :

File: i18n/fr.po [hunk 4]
```diff
@@ -0,0 +1,615 @@
...
+
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerInformationTab.java:63
+msgid "Path:"
+msgstr "Chemin:"
+
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/containers/WinePrefixContainerInformationTab.java:68
+msgid "Installed shortcuts:"
+msgstr "Raccourcis installés:"
```
Comment (qparis) [inline i18n/fr.po]: Please add a space before :

File: i18n/fr.po [hunk 5]
```diff
@@ -0,0 +1,615 @@
...
+
+#: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java:133
+#, java-format
+msgid "This wizard will help you install {0} on your computer"
+msgstr "Cet assistant va vous aider à installer {0} sur votre ordinateur"
+
+#: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java:134
+#, java-format
+msgid "This program was created by: {0}"
+msgstr "Ce programme a été créé par: {0}"
```
Comment (qparis) [inline i18n/fr.po]: Please add a space before :

File: i18n/fr.po [hunk 6]
```diff
@@ -0,0 +1,615 @@
...
+
+#: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java:135
+msgid "For more information about this program, visit:"
+msgstr "Pour plus d'information à propos de ce programme, rendez-vous sur:"
+
+#: phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java:137
+#, java-format
+msgid "This installation program is provided by: {0}"
+msgstr "Cet installeur est fourni par: {0}"
```
Comment (qparis) [inline i18n/fr.po]: Please add a space before :

File: i18n/fr.po [hunk 7]
```diff
@@ -0,0 +1,615 @@
...
+msgstr "Tous"
+
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/LibrarySidebar.java:167
+msgid "Run a script"
+msgstr "Exécuter un script"
+
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/library/LibrarySidebar.java:170
+#, java-format
+msgid "{0} console"
+msgstr "{0} console"
```
Comment (qparis) [inline i18n/fr.po]: Console {0}

File: i18n/fr.po [hunk 8]
```diff
@@ -0,0 +1,615 @@
...
+msgstr "Installé"
+
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/engines/EngineSidebar.java:120
+msgid "Not installed"
+msgstr "Non installé"
+
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/engines/EnginePanel.java:61
+#: phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/settings/AboutPanel.java:72
+msgid "Version:"
+msgstr "Version:"
```
Comment (qparis) [inline i18n/fr.po]: Please add a space before :

**Comment by ImperatorS79** [comment]
Do you want me to add a space before every : or only those you mention ?

**Comment by qparis** [comment]
Before every :)

**Comment by ImperatorS79** [comment]
It's done

**Comment by madoar** [comment]
@ImperatorS79 do you want to give the script translations a try too?

**Comment by ImperatorS79** [comment]
What do you mean ?

**Comment by madoar** [comment]
The scripts contain some strings, like the name of the script, or a description.
These strings can also be translated.
Currently we this has been done for the german language, so if you want you can also do it for french :)
If you're interested take a look here: https://github.com/PlayOnLinux/Scripts/tree/master/i18n

**Comment by madoar** [comment]
Ok to test

**Comment by ImperatorS79** [comment]
Ok, I'll look at that maybe tomorrow

**Pull request merged**